AutoInsurance-Agent/packages/claim/insurance.py

```
@@ -0,0 +1,200 @@
+            PolicyQueries,
+            prompt,
+            claim_info=ev.claim_info.model_dump_json()
+        )
+        return PolicyQueryEvent(queries=queries)
+
+    @step
+    async def retrieve_policy_text(self, ctx: Context, ev: PolicyQueryEvent) -> PolicyMatchedEvent:
+        if self._verbose:
+            ctx.write_event_to_stream(LogEvent(msg=">> Retrieving policy sections"))
+
+        claim_info = await ctx.get("claim_info")
```

Ruff (0.8.2)

Local variable `claim_info` is assigned to but never used

Remove assignment to unused variable `claim_info`

(F841)